packages/typespec-client-generator-core/src/types.ts

```diff
@@ -1867,8 +1868,11 @@ export function handleAllTypes(context: TCGCContext): [void, readonly Diagnostic
     }
   }
   // update for orphan models/enums/unions
-  for (const client of listClients(context)) {
-    const namespaces = [client.service];
+  const allNamespaces = [...context.program.getGlobalNamespaceType().namespaces.values()].filter(
+    (x) => getLocationContext(context.program, x).type === "project",
```

Comment:
this apparently is causing a regression when we are bumping the new version of typespec and TCGC.
Now all orphan models are somehow removed.